.github/workflows/run-tests.yml

```diff
@@ -48,3 +48,43 @@ jobs:
 
     - name: Execute Unit Tests
       run: composer test-ci
+
+  php-windows-tests:
```

Comment:
Well, _that_ is really nice!

Unless I miss something, it's essentially the same as `php-tests:` except:
- `runs-on` (duh!)
- `Set git to use LF` 😬
- `fileinfo` PHP extension


I would also agree we don't need to run it on all variations, so we could just add this via an [`include:`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations) and have a condition for the `Set git to use LF` (we can just always add `fileinfo`, don't see much trouble with it).

This reduces issue when we adapt the flow and having to think it's duplicated for the most parts.

What do you think?

Reply:
How would that work exactly? I'm new to these actions so I just did some copy/paste until it worked.

Reply:
Here's an example on https://github.community/t/create-matrix-with-multiple-os-and-env-for-each-one/16895/2 if you want to try it.

Though in this case `windows-2019` would **not** be in the initial matrix, _only_ in the (one) `include:` directive.

Regarding the condition: it's possible to use (limited) `if` condition on steps, see:
- https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif
- random blog with samples 😄 -> https://dev.to/tomassirio/if-statements-on-github-actions-545d

Reply:
I've merged the jobs. They seem to pass, but I don't see the different OS settings reflected (in the name for example) 🤔

Reply:
I think I've got it all working the way we want.